cos-gpu-installer-docker/entrypoint.sh

```diff
@@ -99,6 +102,12 @@ load_etc_os_release() {
   info "Running on COS build id ${BUILD_ID}"
 }
 
+# Set COS Internal GCS Bucket.
+set_cos_internal_gcs_bucket() {
+  COS_INTERNAL_DOWNLOAD_GCS="https://storage.googleapis.com/container-vm-image-staging/lakitu-release/R${VERSION_ID}-${BUILD_ID}"
+  info "COS internal gcs bucket ${COS_INTERNAL_DOWNLOAD_GCS}"
+}
```

Comment:
I remember that there is a guide line that we should not share internal links on public repos (yes, even when the internal resource is guided by ACL).
Maybe we should consider to not hard code this path here, but add a environment variable `FALLBACK_GCS_PATH` or something, and assign it at runtime.
This way the internal link won't show up on public repo.

(trying to find a reference to that open source contributing guide

Reply:
I have a hard time finding that reference. Maybe I was wrong and this might be just a rumor. Let's not create overhead for ourselves until we see concrete guideline/policy on this. Resolving.